src/ledger/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * an accounting reminder:
+ * https://en.wikipedia.org/wiki/Double-entry_bookkeeping
+ */
+
+import * as accounts from "./accounts"
```

Comment:
`accounts` is plural for the filename. `query` and `transaction` are singular.

Reply:
I didnt want to change the `transaction` to transactios because will make it more difficult to review the changes... maybe we can do it in the renaming refactor or.. let me know and I will rename transaction and query

Reply:
ok you just keep the original name here? let's keep it that way then for now

Reply:
ok